src/pkg/build/conveyorPacker_docker_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package build
+
+import (
+	"testing"
+
+	"github.com/singularityware/singularity/src/pkg/sylog"
+)
+
+// TestPull tests if we can pull an ubuntu image from dockerhub
+func TestConveyor(t *testing.T) {
+	def, err := NewDefinitionFromURI("docker://centos")
+	if err != nil {
+		sylog.Fatalf("unable to parse URI docker://ubuntu:18.04: %v\n", err)
```

Comment:
Doesn't match the URI that was pulled (centos)... also consider const for these?